tests/attr/test_lime.py

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env python3
 
 import unittest
-from typing import Any, Callable, Tuple, Union
+from typing import Any, Callable, Generator, Tuple, Union
```

Comment:
A question about `alt_perturb_func`: in case of `default_perturb_func` we return only the probs but in alternative one it looks like we return masked values.  I might be missing something. I was curious.

Reply:
This is based on the perturb_interpretable_space option, for the default, it is set to True, while for alt_perturb_func, we use perturb_interpretable_space=False.